scripts/redshift/loader.py

```diff
@@ -96,6 +145,57 @@ def load(args):
     _verify_load(es_query=dss_query)
 
 
+def _generate_metadata_schema_version_clause(schema_name: MetadataSchemaName) -> dict:
+    """
+    Generates an ES query clause that selects all supported schema versions for schema_name
+    :param schema_name: MetadataSchemaName to generate clause for
+    :return: dict ES query clause
+    """
+    return {
+        "bool": {
+            "should": [
+                {
+                    "bool": {
+                        "must_not": {
+                            "exists": {
```

Comment:
Maybe we can add some comments around these? To me (with not much ES knowledge), this looks like we don't want bundles that have the `schema_major_version` field. Does that mean we would accept bundles without the schema version annotated this way?

Reply:
Sure, will add comments. 

Yes, specifically, this `must_not` `exists` is in an "OR" (`should`) clause that will match bundles that either 1) have `schema_*_version` fields defined and is within our supported version range or 2) do not have `schema_*_version` fields defined. 

The importance of (2) is to continue to match pre-schema-version bundles before existing bundles are backfilled with the new schema fields. This shouldn't be an issue as we currently support these data.